custom_components/flair/climate.py

```diff
@@ -903,5 +906,8 @@ def set_attributes(setting: str, value: Any, auto_mode: bool) -> dict[str, Any]:
                 attributes ={
                     "swing": value,
                 }
-
+        if setting == 'power':
```

Comment:
Putting this here felt a bit weird, since the return value is always the same regardless of `auto_mode`, but I wanted to follow the existing conventions of the code. Happy to just inline in `async_set_hvac_mode` if you prefer that.

Reply:
Controlling power is never an option for Flair structures that are in Auto mode, which is why you'd want to safeguard against it.